core/operation/src/test/java/uk/gov/gchq/gaffer/named/operation/AddNamedOperationTest.java

```diff
@@ -55,7 +55,12 @@ public void shouldJSONSerialiseAndDeserialise() throws SerialisationException, J
                 .build();
 
         // When
-        String json = new String(JSONSerialiser.serialise(addNamedOperation, true));
+        String json = null;
+        try {
```

Comment:
Can we use the fromJson and toJson methods that are part of the JSONSerialisationTest class.  Similar to how it is written within ForEachTest.  This avoids the need for the SerialisationException bits.

Reply:
Updated to use those methods